### PR TITLE
Change newsbox style, Closes #290

### LINKED ISSUE #290: News box updates
- change title to News & Opinions
- move news above get involved
- use left side for video thumbnail. use a mozfest talk. [design required] [mobile]
- let's consider condensing items on right to more of a bulleted list, so we can list 5-ish things, rather than big font titles of 2-3. 

Estimate includes design and engineering

### PR DESCRIPTION
**Pull request by alanmoo**

### CODE REVIEW

File: source/pug/views/home.pug [hunk 1]
```diff
@@ -2,15 +2,18 @@ extends ../templates/master.pug
...
   .news-item
     .d-flex.align-items-center.mb-3
       img.mr-2.glyph(src=data.glyph)
       p.h6-gray.mb-0=data.outlet
-    h5
-      a.h4-medium-black(href=`${data.link}`)=data.headline
+    h5.mb-2
+      if featured
+        a.h4-medium-black(href=`${data.link}`)=data.headline
+      if !featured
```
Comment (gvn) [inline source/pug/views/home.pug]: Just use `else`?

File: source/pug/views/home.pug [hunk 2]
```diff
@@ -26,6 +29,44 @@ block content
...
+          featured = {
+            headline: "The Algorithmic Spiral of Science",
+            link: "https://www.youtube.com/watch?v=pO-brBVRyN8&index=12&list=PLYiaJo7rYNXLQSEAa2RdyyiS28Ke2Rl60",
+            outlet: "Youtube",
+            author: "Mozilla",
+            date: "February 2017",
+            glyph: `${env.virtualRoot}_images/news-icons/youtube.svg`
+          }
+        .play-button-wrapper
+          img(src='https://assets.mofoprod.net/network-pulse/images/entries/153.jpg')
```
Comment (gvn) [inline source/pug/views/home.pug]: Let's put this image in this repo for now since this item is hardcoded.

File: source/pug/views/home.pug [hunk 3]
```diff
@@ -26,6 +29,44 @@ block content
...
+      .col
+        .mb-5.slanty-background
+          h4.h2-headings-white News & Opinions
+    -
+      data.news = data.news.filter((item) => {
+        return item.featured;
+      });
+    .row.mb-3
+      .col-md-6.mb-3.pb-5
+        //- Hardcoding this next bit because the api doesn't have a thumbnail key yet, and this will always be a video (at least for now)
```
Comment (gvn) [inline source/pug/views/home.pug]: Can you file a follow-on ticket, if there isn't one already, to un-hardcode this?

Comment (alanmoo) [inline source/pug/views/home.pug]: Other changes made, also created https://github.com/mozilla/network/issues/328

**Pull request merged**